src/app/components/Item/ItemsContainer.jsx

```diff
@@ -229,4 +258,13 @@ ItemsContainer.contextTypes = {
   router: PropTypes.object,
 };
 
-export default ItemsContainer;
+const mapStateToProps = state => ({
+  bib: state.bib,
+});
+
+export default {
```

Comment:
Yep this is just so we can test the unwrapped component. I saw this suggested online as a good way to avoid mucking around with `Provider` in the tests. So far I think it makes things easier but we could discuss the best way to test connected components.

The `appConfig` kind of uses the same export style, except the object is named.